### ISSUE TITLE
problems building v2 on latest commit

### ISSUE THREAD

**Issue by brennantaylor** [dsoprea/go-exif#41]
Was working off of https://github.com/dsoprea/go-exif/blob/master/v2/exif-read-tool/main.go for my own thing and I can't get the v2 package to compile. I'm using go 1.14 and go modules.

```
# github.com/dsoprea/go-exif/v2
../../go/pkg/mod/github.com/dsoprea/go-exif/v2@v2.0.0-20200610202304-82fe0e20b97d/ifd_enumerate.go:1502:7: undefined: i
```

**Comment by brennantaylor** [comment]
`7edf52b8857faa2bfa4924621d8d82d7585842c6` seems to work fine

**Comment by dsoprea** [comment]
Yeah. That's on me. I fixed an issue earlier where I was masking that variable with another variable of the same name. I removed that, but probably didn't update the assignment operator, and then didn't audit the tests.

**Comment by dsoprea** [comment]
Fixes are in bd6037a0b8a73ecf405ca813c14fc0e9dc1d7261
